www/include/Administration/parameters/ldap/form.php

```diff
@@ -95,26 +95,33 @@
 );
 $form->addGroup($ldapUseDns, 'ldap_srv_dns', _("Use service DNS"), '&nbsp;');
 
-$ldapDnsUseSsl[] = $form->createElement('radio', 'ldap_dns_use_ssl', null, _("Yes"), '1');
-$ldapDnsUseSsl[] = $form->createElement('radio', 'ldap_dns_use_ssl', null, _("No"), '0');
-$form->addGroup($ldapDnsUseSsl, 'ldap_dns_use_ssl', _("Use SSL connection"), '&nbsp;');
-$ldapDnsUseTls[] = $form->createElement('radio', 'ldap_dns_use_tls', null, _("Yes"), '1');
-$ldapDnsUseTls[] = $form->createElement('radio', 'ldap_dns_use_tls', null, _("No"), '0');
-$form->addGroup($ldapDnsUseTls, 'ldap_dns_use_tls', _("Use TLS connection"), '&nbsp;');
 $form->addElement('text', 'ldap_dns_use_domain', _("Alternative domain for ldap"), $attrsText);
 
 $form->addElement('text', 'ldap_search_limit', _('LDAP search size limit'), $attrsText2);
 $form->addElement('text', 'ldap_search_timeout', _('LDAP search timeout'), $attrsText2);
 
 $query = "SELECT contact_id, contact_name FROM contact WHERE contact_register = '0'";
 $res = $pearDB->query($query);
-$tmplList = array();
-while ($row = $res->fetchRow()) {
-    $tmplList[$row['contact_id']] = $row['contact_name'];
+$tplList = array();
```

Comment:
What tpl means ?
It was "tmp" instead of "tpml" right ?
+ 
We dont know what is the semantic of elements manipulated in this list. We dont realy need to know this is a List type. Better practice is to use this syntaxe : 
(e.g.) 
contacts
foundContacts
unregistredContacts
... (depending of semantic)

Reply:
it's the LDAP user's contact template. As tpl was used almost everywhere in the code, I thought it will be clearer than tmplList ;)